packages/client/src/context/group.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+export interface GroupContextValue {
+  id: string;
+  name: string;
+  displayName: string;
+}
+
+
+export const GroupContext = React.createContext<GroupContextValue>(undefined);
```

Comment:
I thought we should add a function for change context in GroupContext then we can easily integrate GroupContextSelector for change group context value.
ref: https://zh-hant.reactjs.org/docs/context.html#updating-context-from-a-nested-component

Reply:
After discussion, we will determine the final group context  value in [ch11511](https://app.clubhouse.io/infuseai/story/11511/integrate-group-context-aware-components-to-new-ui)